core/src/com/mygdx/insanerpg/model/item/stackable/consumable/gem/WeaponGemSecondaryAttribute.java

```diff
@@ -0,0 +1,27 @@
+package com.mygdx.insanerpg.model.item.stackable.consumable.gem;
+
+import java.util.Random;
+
+public enum WeaponGemSecondaryAttribute {
```

Comment:
Might need to balance min/max bonus since 5% crit chance I think will be underwhelming vs everything else kinda

Reply:
i let it range from 1-3%, with 5 gems on a weapon all EPIC grade that  could give 30% crit which is OP